docs/02-formalia/titlepage.rst

```diff
@@ -117,6 +117,9 @@ Main Features
   pluggable security at three levels: authentication of remote participants, access control of entities and encryption
   of data.
 
+* **Statistics Module.** *eProsima Fast DDS* can be configured to gather and provide information about the data being
```

Comment:
I have not added the link because there is not a link in most of the other features (for example, there is no link to the Security module even though it would also make sense). I do not know if this is something intended.